datafusion/core/src/lib.rs

```diff
@@ -406,19 +406,172 @@
 //! [DataFusion paper submitted SIGMOD]: https://github.com/apache/datafusion/files/13874720/DataFusion_Query_Engine___SIGMOD_2024.pdf
```

Comment:
```suggestion
//! [DataFusion paper submitted to SIGMOD]: https://github.com/apache/datafusion/files/13874720/DataFusion_Query_Engine___SIGMOD_2024.pdf
```
It's not correctly linked.